cmd/account/verify-secrets.go

```diff
@@ -149,15 +169,22 @@ func (o *checkSecretsOptions) run() error {
 		awsClient, err = awsprovider.NewAwsClientWithInput(cred.awsCreds)
 		if err != nil {
 			fmt.Fprintf(o.IOStreams.Out, "Failed to create AWS client with secret %s\n", cred.secret)
-			continue
+			if o.all {
+				continue
+			}
+			return err
 		}
 		if _, err := awsClient.GetCallerIdentity(
 			&sts.GetCallerIdentityInput{}); err != nil {
 			fmt.Fprintf(o.IOStreams.Out, "Failed to get caller identity with secret %s\n", cred.secret)
-			continue
+			if o.all {
+				continue
+			}
+			return err
 		}
 	}
 
+	fmt.Fprintf(o.IOStreams.Out, "Credentials valid for %s\n", o.accountName)
 	return nil
```

Comment:
If we're checking for "all" should we return an err if _any_ of them errored?

Reply:
I'm not sure 100% what to return here, but I realize now that line 187 should be conditional on o.all or not

Reply:
In case `o.all == true` and all creds are valid, could we output a message confirming it?

Reply:
Done